### PR TITLE
Python script test commit

### PR DESCRIPTION
**Pull request by garibaldiviolin**
## This is a PR example

Description

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def ok(abc):
+    return abc + 1
```
Comment (garibaldineto) [inline test.py]:
```suggestion
    return abc + 2
```

File: test.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+def ok(abc):
```
Comment (garibaldineto) [inline test.py:1]: OK

**Review by garibaldineto — COMMENTED** [review]
Suggestion

File: test.py (hunk 2 above)
Comment (garibaldiviolin) [reply test.py:1]: Test this.

Comment (garibaldiviolin) [reply test.py:1]: ok

Comment (garibaldineto) [inline test.py]: ok

**Review by garibaldineto — CHANGES_REQUESTED** [review]
Please change this

File: test.py (hunk 2 above)
Comment (garibaldineto) [inline test.py:1]:
```suggestion
def ok(abc1):
```

**Review by garibaldineto — CHANGES_REQUESTED** [review]
Change that!

**Review by garibaldineto — APPROVED** [review]

**Pull request merged**